MapboxCoreNavigation/RouteController.swift

```diff
@@ -387,7 +391,7 @@ extension RouteController: Router {
         }
         
         let status = status ?? navigator.getStatusForTimestamp(location.timestamp)
-        let offRoute = status.routeState == .offRoute
+        let offRoute = status.routeState == .offRoute || status.routeState == .invalid
```

Comment:
As per discussion with @SiarheiFedartsou in Slack, for now it's better to treat `invalid` state as `offRoute`.